### DISCUSSION TITLE
Can we delete a project?

### DISCUSSION THREAD

**Discussion by GOOGLE-M** [community/community#16268] [Other Feature Feedback, Questions, & Ideas]
I am not sure whether we can delete a project that is not needed any more?

**Comment by MuhammadHamza-Javaid** [comment]
Yes! you can delete any project/ repo. see repo>settings>general
![image](https://user-images.githubusercontent.com/42186984/168141810-62a67426-2c7e-44dc-a06e-c4f418ad6ab1.png)

**Reply by tony-sol** [reply]
> you can delete any project

Yes, but no 😄 
<img width="1047" alt="Снимок экрана 2022-05-14 в 05 00 37" src="https://user-images.githubusercontent.com/33671815/168406765-a8c2032e-4322-46c0-97c6-dc50b78b7649.png">

**Reply by doc-code-hub** [reply]
Maybe it had been locked to changes (archived).

I believe if you reopen it, you should be able to delete it later.

**Reply by tony-sol** [reply]
>if you reopen it, you should be able to delete it later

I wish it would be true, but still 😞 

<img width="1047" alt="2022-05-14 20 06 41" src="https://user-images.githubusercontent.com/33671815/168441716-a38fd40e-3fd7-41bc-bd20-eb641fcd87cd.png">

**Reply by doc-code-hub** [reply]
Are you using an alternative client for Github or the official site? 

I'm asking because i've seen your screenshot that show's a little different UI style and that version number at the bottom (_i don't see any version number_...).
I've never seen nothing like opening and closing project for some "list of open projects"... Maybe i'm missing something...

If you are using some other client, have you tried using the official website github.com instead?

**Reply by tony-sol** [reply]
I'm using only official website or official mobile app for iOS/iPadOS

It's  **"projects (beta)"** actually, not just "projects", so may be thats why we see version number

![Снимок экрана 2022-05-14 в 23 32 09](https://user-images.githubusercontent.com/33671815/168447356-fd2e7cd2-4b69-41fc-8d22-0b638227e05b.png)

**Reply by doc-code-hub** [reply]
Oh, sorry, we (me and [deCoderragnar](https://github.com/deCoderragnar)) was thinking to "project" as repo i guess... 

Actually seem impossible to me too. 

Good Answer. +1

**Reply by tony-sol** [reply]
Actually, some time before there was delete button in projects, but now - its not

Thats pretty strange 'cause i wasn't going to create a project, just clicked on new project to see was there any way to group my repos like gitlab projects

And now i'm stuck with them :smile:

**Comment by doc-code-hub** [comment]
#14121
#11621
#6813
#5436

I found some other duplicated you may want to rise up too :)

This is the comment from timoyates (github collaborator) [comment](https://github.com/github/feedback/discussions/6813#discussioncomment-1545558) _(on 27 Oct 2021)_
